### DISCUSSION TITLE
BitmapLuminanceSource not in version 0.16.8

### DISCUSSION THREAD

**Discussion by lance-ingram** [micjahn/ZXing.Net#420] [Q&A] [answered]
I have upgraded my project to .net version 6.

However, I am getting the following error that BitmapLuminanceSource() doesn't exist in version 0.16.8.

I am unable to find a solution how to resolve this issue. 

Assitance or guidance would be appreciated.

![Screenshot 2022-05-03 at 16 21 36](https://user-images.githubusercontent.com/62103224/166473306-8c2e0359-2ec7-4c72-aaef-13cef2233509.png)

**Comment by micjahn** [comment, marked as answer]
The base package ZXing.Net https://www.nuget.org/packages/ZXing.Net/ only contains the core classes without the implementations for a specific image library. Since .Net Standard / .Net Core the .Net packages don't contain the System.Drawing namespace.
You have to add one of the additional binding packages for the image manipulation library you want to use:
https://www.nuget.org/packages?q=ZXing.Net.Bindings*
Perhaps the package https://www.nuget.org/packages/ZXing.Net.Bindings.Windows.Compatibility/ fits your needs.

**Reply by lance-ingram** [reply]
Yes that resolves the issue thanks @micjahn  for taking time to respond
